helm_deploy/values-dev.yaml

```
@@ -13,6 +13,8 @@ generic-service:
     TOKEN_VERIFICATION_API_URL: "https://token-verification-api-dev.prison.service.justice.gov.uk"
     BREACH_NOTICE_API_URL: "https://breach-notice-api-dev.hmpps.service.justice.gov.uk"
     NDELIUS_INTEGRATION_URL: "https://breach-notice-and-delius-dev.hmpps.service.justice.gov.uk"
+    DELIUS_HOST: "http://localhost"
+    DELIUS_PORT: "7001"
```

You'll need to set this to the correct value in each of the dev, preprod and prod yaml files. The URLs would be: 
* dev: https://ndelius.test.probation.service.justice.gov.uk
* preprod: https://ndelius.pre-prod.delius.probation.hmpps.dsd.io
* prod: https://ndelius.probation.service.justice.gov.uk

Also, it might be simpler to define the full URL here, rather than just the host and port?
```suggestion
    NDELIUS_DEEPLINK_URL: "https://ndelius.test.probation.service.justice.gov.uk/NDelius-war/delius/JSP/deeplink.xhtml"
```